### PR TITLE
Rebased and ready for production 

### PR DESCRIPTION
**Pull request by gosteve2705**

### CODE REVIEW

File: src/models/admin.js [hunk 1]
```diff
@@ -34,13 +35,15 @@ const userSchema = new mongoose.Schema({
   //   default: false
   // },
   resetPasswordToken: String,
+  facebookId: String,
```
Comment (ruzor) [inline src/models/admin.js:38]: Remove oauth from admin model fn

File: src/routes/auth.js [hunk 2]
```diff
@@ -1,15 +1,34 @@
...
   registerValidation,
   loginValidation,
   forgotValidation,
   resetPasswordValidation
 } = require('../utils/validation/joiValidation');
 const { auth } = require('../utils/middleware');
-const { createVerificationLink } = require('../utils/EmailVerification');
+
 const { userForgotPassword, userResetPassword } = require('../controllers/auth');
-const SessionMgt = require('../services/SessionManagement');
```
Comment (ruzor) [inline src/routes/auth.js:12]: Is there a reason why u removed the session management service?

Comment (gosteve2705) [reply src/routes/auth.js:12]: There were 2 instances I only removed one to resolve a conflict

File: src/models/admin.js (hunk 1 above)
Comment (gosteve2705) [reply src/models/admin.js:38]: Ok

Comment (gosteve2705) [reply src/models/admin.js]: Done

File: src/app.js [hunk 3]
```diff
@@ -6,41 +6,83 @@ const cors = require('cors');
...
+
 // configure user session
 SessionMgt.config(app);
 
 // auth routes
 app.use('/api/admin/auth', adminRouter);
 app.use('/api/auth/email', emailVerificationRouter());
 app.use('/api/auth', authorizeUser, userRouter);
+
+app.use('/api/admin/auth/email', emailVerificationRouter());
```
Comment (ruzor) [inline src/app.js:70]: Remove line 70, it was commented out on line 76

Comment (gosteve2705) [reply src/app.js:70]: ok

**Pull request merged**